### ISSUE TITLE
Multiple previous shown editable still shown after new editable activated

### ISSUE THREAD

**Issue by petrabarus** [vitalets/angular-xeditable#267]
I'm experiencing something weird. In the demo page, whenever an editable shown, the other was closed.

But in my case it isn't.

![screenshot from 2015-03-31 14 59 02](https://cloud.githubusercontent.com/assets/523289/6915159/060c0f32-d7bc-11e4-86cf-4c1d7bd6ddbd.png)

So I tried debugging the source code.

The part `e.isDefaultPrevented()` in this [line](https://github.com/vitalets/angular-xeditable/blob/master/dist/js/xeditable.js#L1007)

```
 // bind click to body: cancel|submit|ignore forms
$document.bind('click', function(e) {
    // ignore right/middle button click
    if ((e.which && e.which !== 1) || e.isDefaultPrevented()) {
        return;
    }
    var toCancel = [];
    var toSubmit = [];
```

prevent all other editable closed. 

So I tried removing the `e.isDefaultPrevented()` part, the editables all work fine (i.e. there is only one shown at a moment).

Then I tried debugging to find which part is the culprit by finding who called the `e.preventDefault()`, using [this](http://stackoverflow.com/questions/20490931/how-to-find-what-is-causing-a-preventdefault-which-overides-normal-click-behavio).

```
var oldEPD = Event.prototype.preventDefault;
Event.prototype.preventDefault = function() {
    debugger;
    oldEPD.call(this);
};
```

Here's what I found in the debugger

![screenshot from 2015-03-31 15 47 53](https://cloud.githubusercontent.com/assets/523289/6915361/7de1f8b8-d7bd-11e4-8e41-110becc9a0ab.png)

which is this [line](https://github.com/vitalets/angular-xeditable/blob/master/dist/js/xeditable.js#L950)

Am I doing something wrong? Thanks!

**Comment by Painyjames** [comment]
Have you tried with previous version (0.1.8) ?
Might be a breaking change since e.isDefaultPrevented() has been included recently.
Would be great if you can post a fiddler/plunker so I can have a look as well.

Cheers.

**Comment by petrabarus** [comment]
Yes it's working on 0.1.8.

There is a lot of scripts in my site, so it would take some time to find where the culprit is.

**Comment by fourty2** [comment]
I have the same problem.

**Comment by Rykie** [comment]
I am also running 0.1.8 and I have the exact same issue. When I look on the demo site, it works fine, but as soon as I use it in my site, it does not work. i.e. I can activate multiple inputs to stay in the edit mode, and as soon as I close one, all of them close.

**Comment by klerehag** [comment]
Any plans on fixing this?
It works on the demo site is because it still uses the 0.1.8 version, maybe time to update?

If you take this example http://jsfiddle.net/NfPcH/82/ and switch to using 0.1.9 version then you will get the same behaviour as in this issue. Add this to external resources, https://rawgit.com/vitalets/angular-xeditable/0.1.9/dist/js/xeditable.js

**Comment by rahmatawaludin** [comment]
Hi, I'm experiencing this too. As far I can see, the sample use different controller, right?

**Comment by eugef** [comment]
Fixed by #414
